torch/_inductor/codecache.py

```diff
@@ -1217,6 +1228,17 @@ def pick_vec_isa() -> VecISA:
     if not _valid_vec_isa_list:
         return invalid_vec_isa
 
+    # Set simdlen according to _get_cpu_capability to allow using
+    # the environment ATEN_CPU_CAPABILITY to control CPU vec ISA
+    if (
+        config.cpp.simdlen is None
+        and not isinstance(_valid_vec_isa_list[0], VecNEON)
+        and not isinstance(_valid_vec_isa_list[0], VecZVECTOR)
```

Comment:
`compute_cpu_capability` also handles vsx and zvector. We should also consider them here?

Reply:
Taking VecNEON and VecZVECTOR into account.